Tests/test_mixtures_and_components_combinatorially.py

```diff
@@ -136,7 +198,7 @@ def test_dna_assemblies_in_mixtures(self):
 				for (rbs, args) in self.rbs_classes:
 					R = rbs(**args)
 					for (mixture, args) in self.mixture_classes:
-						A = DNAassembly("assembly", promoter = P, rbs = R)
+						A = DNAassembly("G", promoter = P, rbs = R)
```

Comment:
If you change the first argument to a keyword argument, then we could avoid using the mix of positional and keyword arguments in function calls. (Same suggestion applies to other calls of `DNAassembly` and `Component_CRN_validation`). 
```python
A = DNAassembly(name='G', promoter = P, rbs = R)
```
It would make the code more maintainable in the long run. (Please don't change other function calls, so we can keep the scope of the PR limited to the original issue.) Many Thanks!

Reply:
Done. Will merge now.